pages/toolkits/contribute-a-toolkit.mdx

```diff
@@ -0,0 +1,7 @@
+# How to contribute a toolkit
+
+## Submit your toolkit
+
+To submit your toolkit, please open a pull request on the [Arcade Documentation GitHub repository](https://github.com/ArcadeAI/docs/) that uses the [Community Contributed Toolkit Pull Request Template](https://github.com/ArcadeAI/docs/blob/main/.github/PULL_REQUEST_TEMPLATE/community_contributed_toolkit.md).
```

Comment:
the link to the template will be valid once this PR is merged since the template file is in this PR